addons/web_unsplash/controllers/main.py

```diff
@@ -129,12 +134,16 @@ def fetch_unsplash_images(self, **post):
         access_key = self._get_access_key()
         app_id = self.get_unsplash_app_id()
         if not access_key or not app_id:
+            if not request.env.user._has_unsplash_key_rights():
```

Comment:
The change in `_get_access_key` means that the access_key will always be returned above, and the `_has_unsplash_key_rights` call will only occur when the key isn't set at all. 

@rdeodoo is that a desired change?